pom.xml

```diff
@@ -68,7 +68,7 @@
 		<dependency>
 			<groupId>com.agreemount</groupId>
 			<artifactId>engine</artifactId>
-			<version>1.2.5</version>
+			<version>1.3.0</version>
```

Comment:
Why `1.3.0`? It seems like `1.2.7` is the latest engine version

Reply:
This was done "in advance" ;) The point was corresponding changes are made in 1.3.0 which is not available.